.ci-mgmt.yaml

```diff
@@ -1,6 +1,8 @@
 provider: cloudflare
 major-version: 6
 makeTemplate: bridged
+runner:
+  default: pulumi-ubuntu-8core
```

Comment:
The CloudFlare provider is now failing with `System.IO.IOException: No space left on device.` We are already running with the Free Disk Space option, do we just have to switch to a larger runner?